NICE.Identity.Authentication.Sdk/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using NICE.Identity.Authentication.Sdk.Abstractions;
+using NICE.Identity.Authentication.Sdk.Authentication;
+using NICE.Identity.Authentication.Sdk.Authorisation;
+using NICE.Identity.Authentication.Sdk.Configuration;
+using IConfiguration = Microsoft.Extensions.Configuration.IConfiguration;
+
+namespace NICE.Identity.Authentication.Sdk.Extensions
+{
+	public static class ServiceCollectionExtensions
+	{
+		public static IServiceCollection AddAuthenticationSdk(this IServiceCollection services, IConfiguration configuration)
+		{
+			InstallAuthorisation(services);
+			InstallAuthenticationService(services, configuration);
+			
+			return services;
+		}
+
+		private static void InstallAuthorisation(IServiceCollection services)
```

Comment:
this looks to have reverted the previous code.

see: https://github.com/nhsevidence/identity/blob/master/NICE.Identity.Authentication.Sdk/ServiceCollectionExtensions.cs#L30-L39

Reply:
possibly because you've moved it to an Extensions folder